styles/layouts/_main-navigation.scss

```diff
@@ -436,6 +433,33 @@
         top: 10%;
       }
     }
+    @include touch {
+      $touch-padding-x: 20px;
+      .dropdown-content .search-suggestion-container {
+        .link-item {
+          padding-right: $touch-padding-x;
+          padding-left: $touch-padding-x;
```

Comment:
IMO, it could be shorter `padding: 0 $touch-padding-x;`

Reply:
I didn't want to unnecessary touch vertical padding set elsewhere

Lmk if you still prefer it to be changed

Reply:
OK, Let's keep it.